referenceframe/frame_system_test.go

```diff
@@ -237,3 +237,139 @@ func TestConvertTransformProtobufToFrameSystemPart(t *testing.T) {
 		test.That(t, spatial.OrientationAlmostEqual(part.FrameConfig.pose.Orientation(), testPose.Orientation()), test.ShouldBeTrue)
 	})
 }
+
+func TestFrameSystemToPCD(t *testing.T) {
+	// simplest test
+	fs := NewEmptySimpleFrameSystem("test")
+	// ------
+	name0 := "frame0"
+	pose0 := spatial.NewPoseFromPoint(r3.Vector{-4, -4, -4})
+	dims0 := r3.Vector{2, 2, 2}
+	geomCreator0, err := spatial.NewBox(pose0, dims0, "box0")
+	test.That(t, err, test.ShouldBeNil)
+	frame0, err := NewStaticFrameWithGeometry(name0, pose0, geomCreator0)
+	test.That(t, err, test.ShouldBeNil)
+	fs.AddFrame(frame0, fs.World())
+	// -----
+	name1 := "frame1"
+	pose1 := spatial.NewPoseFromPoint(r3.Vector{2, 2, 2})
+	dims1 := r3.Vector{2, 2, 2}
+	geomCreator1, err := spatial.NewBox(pose1, dims1, "box1")
+	test.That(t, err, test.ShouldBeNil)
+	frame1, err := NewStaticFrameWithGeometry(name1, pose1, geomCreator1)
+	test.That(t, err, test.ShouldBeNil)
+	fs.AddFrame(frame1, frame0)
+
+	inputs := make(map[string][]Input)
```

Comment:
this can go away and be replaced by the utility function we have `input := referenceframe.StartPositions(fs)`